include/boost/beast/core/buffers_generator.hpp

```diff
@@ -0,0 +1,200 @@
+//
+// Copyright (c) 2022 Seth Heeren (sgheeren at gmail dot com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/beast
+//
+
+#ifndef BOOST_BEAST_CORE_BUFFERS_GENERATOR_HPP
+#define BOOST_BEAST_CORE_BUFFERS_GENERATOR_HPP
+
+#include <boost/beast/core/detail/config.hpp>
```

Comment:
I notice you removed the whitespace separating config.hpp from following includes.

I used the separator because it highlights that we want to keep `config` first.

It was meant to prevents tools (both human and machine tools :)) from re-ordering includes so `config` isn't first anymore.

Any merit?